src/main/kotlin/de/bund/digitalservice/useid/config/SecurityConfig.kt

```diff
@@ -25,9 +26,10 @@ class SecurityConfig(
     @Bean
     fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
         return http.authorizeHttpRequests()
-            .requestMatchers(HttpMethod.GET, "$IDENTIFICATIONS_BASE_PATH/*/transaction-info").permitAll()
             .requestMatchers("$IDENTIFICATIONS_BASE_PATH/**").authenticated()
             .requestMatchers("$IDENTIFICATIONS_BASE_PATH/**").hasAuthority(MANAGE_IDENTIFICATION_SESSION_AUTHORITY)
+            .requestMatchers(HttpMethod.POST, "/api/v1/identifications/*/$TRANSACTION_INFO_SUFFIX").authenticated()
+            .requestMatchers(HttpMethod.POST, "/api/v1/identifications/*/$TRANSACTION_INFO_SUFFIX").hasAuthority(MANAGE_IDENTIFICATION_SESSION_AUTHORITY)
```

Comment:
Why do we have to add this? I would have assumed that the pattern in line 29 and 30 would match already also this endpoint. Otherwise, we would change behavior as the only the name of the endpoints has changed. Or am I missing something?

If we keep it, could we use the constant:
```suggestion
            .requestMatchers(HttpMethod.POST, "$IDENTIFICATIONS_BASE_PATH/*/$TRANSACTION_INFO_SUFFIX").authenticated()
            .requestMatchers(HttpMethod.POST, "$IDENTIFICATIONS_BASE_PATH/*/$TRANSACTION_INFO_SUFFIX").hasAuthority(MANAGE_IDENTIFICATION_SESSION_AUTHORITY)
```

Reply:
Talked to @mpanne  and we can remove those lines